PerigonGames/BForBoss
# Cole/BFB-324: Created Enemy Spawn Effects

**JIRA Ticket**
https://perigongames.atlassian.net/browse/BFB-324

**Description**
I created a prefab containing a timeline sequence and animation that controls the spawning effect for the flying enemy. 

To test:
Go to any scene (preferably one that has correct lighting) and drag in Assets > _BForBoss > _Entities > _Enemies > FlyingRobot > P_EnemySpawnEffect.prefab. Then click on the prefab gameObject, and open the Timeline window (Window > Sequencing > Timeline). Press play and observe the effect.

For integration:
It may be useful to use the signal system found in the Sequence Timeline for integrating the timing between the effect and the actual enemy spawning. Enemy should not spawn until the effect has finished, or should be invisible until the effect has finished.

**Screenshots / GIFs**
GIF compression wasn't playing nice. Video can be seen here [https://perigongames.atlassian.net/browse/BFB-324](https://perigongames.atlassian.net/browse/BFB-324) called "SPAWN IN EFFECT.mp4"
